### PR TITLE
Fix path of AirMapModule

### PR DESCRIPTION
**Pull request by spikebrehm**
The PR that added `AirMapModule` (#449) was merged after the PR that
moved all the `lib/src/*` files to just `src/*` (#502). Fix it by moving
`AirMapModule` to `src/`.

cc: @rops @alvinthen

### CODE REVIEW

File: android/src/main/java/com/airbnb/android/react/maps/AirMapManager.java [hunk 1]
```diff
@@ -1,7 +1,6 @@
 package com.airbnb.android.react.maps;
 
 import android.view.View;
-import android.content.Context;
```
Comment (spikebrehm) [inline android/src/main/java/com/airbnb/android/react/maps/AirMapManager.java]: This was an unused import.

**Comment by spikebrehm** [comment]
cc: @felipecsl

**Comment by alvinthen** [comment]
My bad there, thanks for fixing

**Comment by felipecsl** [comment]
👍

**Comment by spikebrehm** [comment]
Just published [0.8.1](https://github.com/lelandrichardson/react-native-maps/releases/tag/v0.8.1) that includes this change.

**Comment by spikebrehm** [comment]
It turns out 0.8.1 was faulty, because my working directory had an additional example app containing the `react-native` node module, which got published as part of the NPM package (:facepalm:), causing a `@providesModule` error, so I've unpublished it and published [0.8.2](https://github.com/lelandrichardson/react-native-maps/releases/tag/v0.8.2).  Use that instead!

**Pull request merged**